diff -Nur jakarta-tomcat-4.1.24-src.orig/catalina/src/share/org/apache/catalina/servlets/HTMLManagerServlet.java jakarta-tomcat-4.1.24-src.chng/catalina/src/share/org/apache/catalina/servlets/HTMLManagerServlet.java
--- jakarta-tomcat-4.1.24-src.orig/catalina/src/share/org/apache/catalina/servlets/HTMLManagerServlet.java	Wed Mar 19 10:20:48 2003
+++ jakarta-tomcat-4.1.24-src.chng/catalina/src/share/org/apache/catalina/servlets/HTMLManagerServlet.java	Wed Jun 25 15:23:52 2003
@@ -84,7 +84,7 @@
 import org.apache.catalina.Host;
 import org.apache.catalina.util.ServerInfo;
 import org.apache.commons.fileupload.FileItem;
-import org.apache.commons.fileupload.FileUpload;
+import org.apache.commons.fileupload.DiskFileUpload;
 import org.apache.commons.fileupload.FileUploadException;
 
 /**
@@ -195,7 +195,7 @@
         String message = "";
 
         // Create a new file upload handler
-        FileUpload upload = new FileUpload();
+        DiskFileUpload upload = new DiskFileUpload();
 
         // Get the tempdir
         File tempdir = (File) getServletContext().getAttribute
@@ -259,7 +259,7 @@
                         ("htmlManagerServlet.installUploadWarExists",war);
                     break;
                 }
-                warUpload.write(file.getCanonicalPath());
+                warUpload.write(file);
                 try {
                     URL url = file.toURL();
                     war = url.toString();
