diff -urN jakarta-tomcat-4.0.6-src.org/catalina/src/bin/catalina.sh jakarta-tomcat-4.0.6-src/catalina/src/bin/catalina.sh
--- jakarta-tomcat-4.0.6-src.org/catalina/src/bin/catalina.sh	Tue Nov 26 22:37:05 2002
+++ jakarta-tomcat-4.0.6-src/catalina/src/bin/catalina.sh	Tue Nov 26 22:52:30 2002
@@ -104,6 +104,10 @@
   JSSE_HOME=`cygpath --path --windows "$JSSE_HOME"`
 fi
 
+if [ -x /usr/lib/java/bin/java ]; then                                                                          
+  JAVA_HOME=/usr/lib/java                                                                                       
+fi
+
 # ----- Execute The Requested Command -----------------------------------------
 
 echo "Using CATALINA_BASE:   $CATALINA_BASE"
diff -urN jakarta-tomcat-4.0.6-src.org/catalina/src/bin/tool-wrapper.sh jakarta-tomcat-4.0.6-src/catalina/src/bin/tool-wrapper.sh
--- jakarta-tomcat-4.0.6-src.org/catalina/src/bin/tool-wrapper.sh	Tue Nov 26 22:37:05 2002
+++ jakarta-tomcat-4.0.6-src/catalina/src/bin/tool-wrapper.sh	Tue Nov 26 22:53:07 2002
@@ -50,6 +50,10 @@
   [ -n "$CLASSPATH" ] && CLASSPATH=`cygpath --path --unix "$CLASSPATH"`
 fi
 
+if [ -x /usr/lib/java/bin/java ]; then   
+      JAVA_HOME=/usr/lib/java
+fi
+  
 # Get standard Java environment variables
 if [ -r "$CATALINA_HOME"/bin/setclasspath.sh ]; then
   BASEDIR="$CATALINA_HOME"
